RELEASES.md

```diff
@@ -18,16 +18,54 @@ Unreleased.
 
 ### Changed
 
+* The `WasiCtxBuilder` type has been refactored, and `WasiCtx` now has private
+  fields.
+  [#6652](https://github.com/bytecodealliance/wasmtime/pull/6652)
+
 ### Fixed
 
 * Only export the top-level preview2 module from wasmtime-wasi when the
   `preview2` feature is enabled.
   [#6615](https://github.com/bytecodealliance/wasmtime/pull/6615)
 
+
+### Cranelift changes
+
+* Tail call implementation has begun in cranelift
+  [#6641](https://github.com/bytecodealliance/wasmtime/pull/6641)
+  [#6666](https://github.com/bytecodealliance/wasmtime/pull/6666)
+  [#6650](https://github.com/bytecodealliance/wasmtime/pull/6650)
+  [#6635](https://github.com/bytecodealliance/wasmtime/pull/6635)
+  [#6608](https://github.com/bytecodealliance/wasmtime/pull/6608)
+  [#6586](https://github.com/bytecodealliance/wasmtime/pull/6586)
+
+* Work continues on SIMD support for the riscv64 backend
+  [#6657](https://github.com/bytecodealliance/wasmtime/pull/6657)
+  [#6643](https://github.com/bytecodealliance/wasmtime/pull/6643)
+  [#6601](https://github.com/bytecodealliance/wasmtime/pull/6601)
+  [#6609](https://github.com/bytecodealliance/wasmtime/pull/6609)
+  [#6602](https://github.com/bytecodealliance/wasmtime/pull/6602)
+  [#6598](https://github.com/bytecodealliance/wasmtime/pull/6598)
+  [#6599](https://github.com/bytecodealliance/wasmtime/pull/6599)
+  [#6587](https://github.com/bytecodealliance/wasmtime/pull/6587)
+  [#6568](https://github.com/bytecodealliance/wasmtime/pull/6568)
+  [#6515](https://github.com/bytecodealliance/wasmtime/pull/6515)
+
+* Fix `AuthenticatedRet` when stack bytes are popped in the aarch64 backend
+  [#6634](https://github.com/bytecodealliance/wasmtime/pull/6634)
+
+* Lower SIMD requirements to SSE2 on x64
```

Comment:
We could say a bit more about this here -- it's a pretty nice accomplishment and something I'm not aware of any other engine supporting on x86 (Wasm-SIMD itself is on record as having been designed with modern SSE4 in mind, e.g. [here](https://github.com/bytecodealliance/wasmtime/pull/6224#issuecomment-1553306884).

Perhaps something like "Wasmtime/Cranelift on x86-64 can now execute Wasm-SIMD on baseline SSE2, which all x86-64 processors support (as part of the base x86-64 spec). Previously, SSE4.2 extensions were required. This new work allows Wasm with SIMD extensions to execute on processors produced back to 2003."

Reply:
I like your suggestion!